libkbfs/conflict_resolver.go

```diff
@@ -3043,6 +3066,12 @@ func (cr *ConflictResolver) completeResolution(ctx context.Context,
 		return err
 	}
 
+	defer func() {
```

Comment:
Never mind, I see why this is needed now. Perhaps a comment explaining that `syncBlocks` also cleans up, but we want to clean up on errors form finalizeResolution too?